api/src/main/java/org/openmrs/module/idgen/IdentifierPool.java

```diff
@@ -29,11 +29,11 @@
 	//***** PROPERTIES *****
 	
     private IdentifierSource source;
-    private int batchSize = 1000; // for requests to pool
-    private int minPoolSize = 500; // request more when we go below this number
-    private boolean sequential = false;
+    private Integer batchSize = 1000; // for requests to pool
+    private Integer minPoolSize = 500; // request more when we go below this number
+    private Boolean sequential = false;
     private Set<PooledIdentifier> identifiers;
-    private boolean refillWithScheduledTask = true; // if true, refill with a scheduled task, otherwise refill on demands for ids
+    private Boolean refillWithScheduledTask = true; // if true, refill with a scheduled task, otherwise refill on demands for ids
 
```

Comment:
I don't have any problem with these changes in theory, but I'm not 100% confident that the code that references them is null-safe or that suitable defaults will be set.  Is this something you have addressed?